llvm/lib/Transforms/IPO/GlobalOpt.cpp

```diff
@@ -1610,9 +1610,9 @@ optimizeOnceStoredGlobal(GlobalVariable *GV, Value *StoredOnceVal,
           nullptr /* F */,
           GV->getInitializer()->getType()->getPointerAddressSpace())) {
     if (Constant *SOVC = dyn_cast<Constant>(StoredOnceVal)) {
-      if (GV->getInitializer()->getType() != SOVC->getType())
+      if (GV->getInitializer()->getType() != SOVC->getType()) {
         SOVC = ConstantExpr::getBitCast(SOVC, GV->getInitializer()->getType());
-
+      }
```

Comment:
Please, commit this change to llorg instead of intel/llvm.
This part is not relevant to SYCL.

Reply:
Removed the file from change set.